README.md

```diff
@@ -629,6 +631,20 @@ Usage: `nerdctl create [OPTIONS] IMAGE [COMMAND] [ARG...]`
 
 The `nerdctl create` command similar to `nerdctl run -d` except the container is never started. You can then use the `nerdctl start <container_id>` command to start the container at any point.
 
+### :whale: nerdctl cp
+Copy files/folders between a running container and the local filesystem
+
+Usage:
+- `nerdctl cp [OPTIONS] CONTAINER:SRC_PATH DEST_PATH|-`
+- `nerdctl cp [OPTIONS] SRC_PATH|- CONTAINER:DEST_PATH`
+
+:warning: This command is not designed to be used with untrusted containers. Unexpected behavior of `nerdctl cp`  are not treated as a vulnerability. Users must be conscious of that.
```

Comment:
cc @dmcgowan @samuelkarp for the security policy

Reply:
I've been hoping to find time to review this PR (and its predecessor) but have been unable to do so.

With that said, maybe we can reword the warning a bit?

> :warning: `nerdctl cp` is designed only for use with trusted, cooperating containers.  Using `nerdctl cp` with untrusted or malicious containers is unsupported and may not provide protection against unexpected behavior.

Reply:
Thanks, reworded